tools/shell-client.ts

```diff
@@ -0,0 +1,221 @@
+import * as assert from "assert";
+import * as fs from "fs";
+import * as path from "path";
+import * as net from "net";
+import { isEmacs } from "./utils/utils";
+//@ts-ignore
```

Comment:
Ignored, because:
```
Could not find a declaration file for module 'chalk'.
'/git/meteor/tools/node_modules/chalk/index.js' implicitly has an 'any' type.
Try `npm install @types/chalk` if it exists or add a new declaration (.d.ts) file containing `declare module 'chalk';`ts(7016)
```

Reply:
chalk 2.2.0+ has type definitions file https://github.com/chalk/chalk/releases/tag/v2.2.0

should we update chalk?

Reply:
I’m going to keep the `require` style for now, per https://github.com/meteor/meteor/pull/10614#discussion_r302156889.